io_scene_nif/modules/animation/animation_import.py

```diff
@@ -87,10 +87,11 @@ def create_action(b_obj, action_name):
         # could probably skip this test and create always
         if not b_obj.animation_data:
             b_obj.animation_data_create()
-        if action_name in bpy.data.actions:
-            b_action = bpy.data.actions[action_name]
-        else:
-            b_action = bpy.data.actions.new(action_name)
+        # do not retrieve
```

Comment:
Remove is no longer needed.

Reply:
Actually I'll have to double check the other use cases. Maybe we should add a toggle param for this.
For nif import we _probably_ need to retrieve the action of that name to avoid creating dozens of copies of the anim, each with one channel for the property that is animated.

Reply:
Yeah, retrieve is needed in other cases. I just made retrieve the default and can optionally turn it off with a toggle.

Reply:
Cool, but I don't seen any calls to the method where retrieve=False is set.

Reply:
Nope, just me missing it - https://github.com/niftools/blender_nif_plugin/pull/311/files#diff-1e3ea715dc26bebefefdd235d03506d9R89